C1C0_path_planning.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Get rid of this file, should not be here.

Reply:
Amen